images/scripts/rhel4edge.setup

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+set -eux
+
+podman pull quay.io/cockpit/ws
+podman pull quay.io/jitesoft/nginx
+# for c-podman tests
+/var/lib/testvm/podman-images.setup
+
+# Prevent SSH from hanging for a long time when no external network access
+echo 'UseDNS no' >> /etc/ssh/sshd_config.d/10-no-usedns.conf
+
+# Admin is reserved name and IB does not allow creating it
+if ! getent passwd admin >/dev/null; then
+    useradd -c Administrator -G wheel admin
+    echo foobar | passwd --stdin admin
+fi
```

Comment:
This isn't going to work with the already existing "admin" user created in the request JSON above. Either we can set password and wheel membership in the request as well and drop this, or the `if` needs to be dropped, and changed to `usermod`.

Reply:
I will keep it. If it eventually starts to work as intended this will fail and then we need to change it to usermod as I don't think it is possible to add user to wheel and define password.